Classes/Private/Manager/Audio/OCTAudioEngine.h

```diff
@@ -35,12 +38,14 @@
 - (BOOL)stopAudioFlow:(NSError **)error;
 
 /**
- * Send the audio to the speaker
- * @param speaker YES to send audio to speaker, NO to reset to default.
- * @param error Pointer to error object.
- * @return YES if successful override, otherwise NO.
+ * Set the input device (not available on Mac OS X).
```

Comment:
Even better - maybe we should make whole method `setInputDeviceID:error:` OS X only? That way there would be no need for this comment, for exception for iOS